### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Disk_CreateSnapshotDisk_ScreenReader: NVDA does not read as header for the Create Snapshot.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2069]
" [Check out Accessibility Insights! ](https://accessibilityinsights.io/)Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA;#JAWS; #Win32;

Environment Details:
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.1
NVDA Version:2019.2
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to CreateSnapshot button and click on it.
7. "CreateSnapshot" Popup gets opened.
8.Start NVDA and using Tab key and check whether the NVDA reads as header for Create Snapshot.
**Actual:​**
NVDA does not read as header for the Create Snapshot.

**Same issue is observed throughout the application where these type of dialogs present.**
Same issue is observed in JAWS also.
**Expected:**
NVDA should read as header for the Create Snapshot .

**User Impact:**
Screen reader users unable to know Create Snapshot as header.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[2069_A11y_AzureToolsStorageExplorer_Disk_CreateSnapshotDisk_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3720682/2069_A11y_AzureToolsStorageExplorer_Disk_CreateSnapshotDisk_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please elaborate on what the issue is? You open the dialog, NVDA announces the title of the window, and then that's it.  If you had hit tab, focus would've gone to the instructions, which would've been read, and then if you hit tab again, the text box would have been focused. What else are you looking for?

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please follow up?

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please follow up?

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please follow up?

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT
When the dialog opens, the title is properly announced by screenreader.
 But,here the issue is that the text "Create Snapshot" should be announced as a header element when screenreader user navigate to that text.

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo , are you expecting the header to be read because the input does not have keyboard focus/because it is not clear where the focus is? What if initially after opening, the text box had focus, and then NVDA read/described that input. Would that solve your complaint?

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT NVDA focus should go by arrow key and it should read as header word.Keyboard focus should not go to that header as non text content.

**Comment by MRayermannMSFT** [comment]
Are you looking for NVDA to say something like:
```
heading level 1
Create Snapshot
```

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo I've gone ahead and made a change that does as I proposed. I'm going to close this bug now. Please email or ping me on teams if that change is not what you were looking for.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and the issue is no longer reproduced,hence close this issue.
![image](https://user-images.githubusercontent.com/53940787/73659930-3890d300-46bd-11ea-8020-a9a516ffa73c.png)

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
